data/enderio/versions/2.3.0.424_beta/mod-version.cg

```diff
@@ -934,4 +934,4 @@ group: Tools
         recipe:
             input: Basic Gear, Electrical Steel
             pattern: 1.1 .0. .1.
-            tools: Crafting Table
\ No newline at end of file
+            tools: Crafting Table# item: Basic Capacitor Bank_2
```

Comment:
I don't know either. I only loaded in new textures for the EnderIO mod. Might be some error I made by accident, I'll revert it back.